modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-data-money/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+const grantNumber = Number(
```

Comment:
Space

```suggestion

const grantNumber = Number(
```

Reply:
the variable name can be: 

```suggestion
const grantMoney = Number(
```

Important: Variables names should have clear name, see the example

https://github.com/ryanmcdermott/clean-code-javascript#use-meaningful-and-pronounceable-variable-names